warehouse-management-service/internal/handler/handler_test.go

```diff
@@ -861,23 +883,29 @@ func TestUpdateShelfBlock(t *testing.T) {
 func TestUpdateShelfBlockRequestError(t *testing.T) {
 	tests := []struct {
 		updateWarehouseRequest interface{}
-		want                   api.ShelfBlockResponse
+		want                   []string
 		wantStatusCode         int
 	}{
 		{
 			updateWarehouseRequest: nil,
-			want:                   api.ShelfBlockResponse{Error: "request body cannot be empty"},
+			want:                   []string{"request body cannot be empty"},
 			wantStatusCode:         http.StatusBadRequest,
 		},
 		{
 			updateWarehouseRequest: map[string]string{"foo": "bar"},
-			want:                   api.ShelfBlockResponse{Error: "Failed to parse request"},
+			want:                   []string{"Failed to parse request"},
 			wantStatusCode:         http.StatusBadRequest,
 		},
 		{
 			updateWarehouseRequest: api.UpdateShelfBlockRequest{},
-			want:                   api.ShelfBlockResponse{Error: "Invalid input: id cannot be empty, aisle cannot be empty, rack cannot be empty, storageType cannot be empty, warehouseId cannot be empty"},
-			wantStatusCode:         http.StatusBadRequest,
+			want: []string{
```

Comment:
Generally avoid top level arrays in favour of wrapping them in a map. In this case, it can be keyed by "errors". Will keep the response structure open for extension.

Reply:
Tried making this a `map[string]interface{}` to allow for extending the response structure. But this makes reading from the errors map value a bit cluttered, and needs casting to `[]string`. 

Is renaming `want` to `errors` better? That way, we could use `want` for other response types, that we may want to extend to.